DigitalLearningSolutions.Data/Models/CustomPrompts/CustomPrompt.cs

```diff
@@ -1,16 +1,30 @@
 ﻿namespace DigitalLearningSolutions.Data.Models.CustomPrompts
 {
+    using System.Collections.Generic;
+
     public class CustomPrompt
     {
         public CustomPrompt(string? text, string? options, bool mandatory)
         {
             CustomPromptText = text;
-            Options = options;
+            Options = SplitOptionsString(options);
             Mandatory = mandatory;
         }
-        
+
+        private List<string> SplitOptionsString(string? options)
+        {
+            var optionsList = new List<string>();
+            if (options != null)
+            {
+                optionsList.AddRange(options.Split(new [] { '\r', '\n' },
+                    System.StringSplitOptions.RemoveEmptyEntries));
+            }
+
+            return optionsList;
```

Comment:
I've noticed that the data test for this checks the default mapping, but I'm wondering if we want to write a few unit tests just for this model to make sure the mapping in the constructor works as expected? (e.g. options = null, one item in a list, and a case with several options)